spring-petclinic-test/src/main/java/com/griddynamics/web/selenium/CustomPropertyRemoteWebDriverProvider.java

```diff
@@ -0,0 +1,45 @@
+package com.griddynamics.web.selenium;
+
+import org.jbehave.web.selenium.RemoteWebDriverProvider;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxProfile;
+
+public class CustomPropertyRemoteWebDriverProvider extends RemoteWebDriverProvider {
+
+    @Override
+    public CustomPropertyRemoteWebDriverProvider(DesiredCapabilities desiredCapabilities) {
+      super(desiredCapabilities);
+
+        FirefoxProfile profile = new FirefoxProfile();
+
+//        profile.setPreference("app.update.enabled", "false");
+//        profile.setPreference("app.update.auto", "false");
+//        profile.setPreference("browser.search.update", "false");
+//        profile.setPreference("extensions.update.enabled", "false");
+        profile.setPreference("app.update.checkInstallTime", "false");
+        profile.setPreference("extensions.update.autoUpdateDefault", "false");
+        profile.setPreference("lightweightThemes.update.enabled", "false");
+        
+        desiredCapabilities.setCapability(FirefoxDriver.PROFILE, profile);
+        super(desiredCapabilities);
```

Comment:
Compilation failure:
[ERROR] /home/kevstigneev/proj/ASF/asf-webapp-demo/spring-petclinic-test/src/main/java/com/griddynamics/web/selenium/CustomPropertyRemoteWebDriverProvider.java:[25,13] error: call to super must be first statement in constructor